packages/packagers/js/src/index.js

```diff
@@ -62,6 +62,11 @@ export default (new Packager({
       ({contents, map} = await packager.package());
     }
 
+    if (bundle.env.sourceType === 'module') {
+      contents = "'use strict';\n" + contents;
+      map?.offsetLines(1, 1);
```

Comment:
Maybe we can avoid this (offsetting large source maps can be slow) if we add this within the individual packagers themselves before we add any content? Also I think for the ScopeHoistingPackager we only need this if the output format is not ESM?